### PR TITLE
Mention composite builds in the configuration cache release notes

### PR DESCRIPTION
**Pull request by bamboo**
And update the configuration cache docs.

### CODE REVIEW

**Comment by bamboo** [comment]
@bot-gradle test this

**Review by eskatos — APPROVED** [review]
🎉

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ See [the userguide](userguide/more_about_tasks.html#sec:property_file_normalizat
 
 [//]: # (TODO context and overview of improvements)
 
+#### Composite builds
+
+Starting with this release, [composite builds](userguide/composite_builds.html) are fully supported with the configuration cache and thus will no longer be reported as problems.
```
Comment (pioterj) [inline subprojects/docs/src/docs/release/notes.md:99]: "will no longer be reported as problems" sounds a bit weird to me. I think just removing this part of the sentence would be fine.

**Review by pioterj — CHANGES_REQUESTED** [review]

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (bamboo) [reply subprojects/docs/src/docs/release/notes.md]: Thanks for the feedback, @pioterj. Done.

**Pull request merged**